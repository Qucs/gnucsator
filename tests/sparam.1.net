

Pac:P2 _net0 gnd Num="2" Z="50 Ohm" P="1" f="1 GHz" Temp="26.85"
Pac:P1 _net2 gnd Num="1" Z="50 Ohm" P="100" f="1 GHz" Temp="26.85"
R:R3 _net1 _net0 R="1 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
C:C1 _net1 _net0 C=".01"
R:RR gnd _net1 R="1 Ohm" Temp="" Tc1="0.0" Tc2="0.0" Tnom="26.85"
.SP:SP1 Type="lin" Start="1" Stop="1e1" Points="10" Noise="no" NoiseIP="1" NoiseOP="2" saveCVs="no" saveAll="no"
R:R2 _net2 _net1 R="1 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
C:C2 _net1 _net2 C=".1"
VCVS:SRC1 _net2 _net3 gnd gnd G="1" T="0"
R:RR1 _net0 _net3 R="1 Ohm" Temp="" Tc1="0.0" Tc2="0.0" Tnom="26.85"
