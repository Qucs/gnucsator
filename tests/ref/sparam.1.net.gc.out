<Qucs Dataset>
default plugins: ignored
detach_all: already installed, replacing
stashing as detach_all:0
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
spice_bjt: already installed, replacing
stashing as spice_bjt:0
spice_mos: already installed, replacing
stashing as spice_mos:0
R:RR gnd _net1 R="1 Ohm" Temp="" Tc1="0.0" Tc2="0.0" Tnom="26.85" 
                         ^ ? RR: Temp has no value?
.SP:SP1 Type="l ... p="1e1" Points="10" Noise="no" NoiseIP="1" NoiseOP="2" s
                                        ^ ? what's this (incomplete)?
R:RR1 _net0 _net3 R="1 Ohm" Temp="" Tc1="0.0" Tc2="0.0" Tnom="26.85" 
                            ^ ? RR1: Temp has no value?
Gnucap   System status
iterations: op=3, dc=0, tran=0, fourier=0, total=13
transient timesteps: accepted=0, rejected=0, total=0
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=4, subckt=2, model=0, total=6
dctran density=61.1%, ac density=61.1%

<indep frequency 10>
 1.
 2.
 3.
 4.
 5.
 6.
 7.
 8.
 9.
 10.
</indep>
<dep S[2,2] frequency>
-0.9614256+ 138.4424E-6* i
-0.9613167+ 142.5208E-6* i
-0.9612606+ 108.0482E-6* i
-0.9612341+ 73.67465E-6* i
-0.9612214+ 44.63062E-6* i
-0.9612158+ 20.28811E-6* i
-0.9612142- 479.0769E-9* i
-0.961215- 18.53722E-6* i
-0.9612175- 34.48256E-6* i
-0.9612212- 48.72034E-6* i
</dep>
<dep S[2,1] frequency>
 0.06486477- 0.01614152* i
 0.0529966- 0.0177851* i
 0.04676392- 0.01525693* i
 0.04364353- 0.0127186* i
 0.04195072- 0.01072787* i
 0.04095164- 0.009211202* i
 0.04031937- 0.008041898* i
 0.0398963- 0.007121964* i
 0.03960032- 0.006383227* i
 0.03938563- 0.005778837* i
</dep>
<dep S[1,2] frequency>
 0.01883797+ 830.0557E-6* i
 0.01906053+ 0.001453945* i
 0.01924141+ 0.002012483* i
 0.01941477+ 0.002560473* i
 0.01960269+ 0.003102552* i
 0.01981359+ 0.003634786* i
 0.02004992+ 0.004152794* i
 0.02031152+ 0.004652924* i
 0.02059697+ 0.005132291* i
 0.02090422+ 0.005588671* i
</dep>
<dep S[1,1] frequency>
-0.9203348- 0.02437192* i
-0.938226- 0.02689305* i
-0.9476264- 0.02312622* i
-0.9523388- 0.01934331* i
-0.9549022- 0.01638473* i
-0.9564223- 0.01413941* i
-0.9573919- 0.01241623* i
-0.9580483- 0.01106732* i
-0.958515- 0.009989804* i
-0.9588608- 0.009112942* i
</dep>
